woocommerce/woocommerce-android
# [Hazmat Shipping] Remove hazmat feature flag and release it

Summary
==========
Effectively remove the Hazmat Shipping feature flag and release the feature to our users.

How to Test
==========
1. Simply execute the same Shipping Label creation steps with a package containing a Hazmat declaration to make sure the entire flow still works.

Update release notes:
